tools/codegen/codegen.bzl

```diff
@@ -33,7 +33,7 @@ codegen = rule(
     output_to_genfiles = True,  # so that header files can be found.
     attrs = {
         "data": attr.label_list(
-            allow_files = [".json", ".yaml"],
+            allow_files = [".json", ".yaml", ".pkgdef"],
```

Comment:
is codegen now also in the enkit repo? if yes, we should switch to using that version.

Reply:
(approving, regardless)